src/acl.js

```diff
@@ -264,22 +264,25 @@ Acl.prototype.add = function(options, callback) {
     query.generation = options.generation;
   }
 
-  this.request({
-    method: 'POST',
-    uri: '',
-    qs: query,
-    json: {
-      entity: options.entity,
-      role: options.role.toUpperCase()
-    }
-  }, function(err, resp) {
-    if (err) {
-      callback(err, null, resp);
-      return;
-    }
+  this.request(
```

Comment:
Is there a style rule that forces this that we can ignore? The way that was removed is generally what I've seen in other codebases. I'd rather factor out the object argument so we can at least end up with:

```js
this.request(reqOpts, function(err, resp) {
  // ...
});
```

Reply:
Making the change you described would get that result. (Prettier tries to maximize what it will fit on a line.) I actually would prefer the code you wrote in your comment over _both_ what was there before _and_ what Prettier spit out. (I agree that what was there before was better than what Prettier spit out.)